src/components/composer/composer/composer.xml

```diff
@@ -9,7 +13,9 @@
         t-att-contenteditable="env.model.getters.isReadonly() ? 'false' : 'true'"
         spellcheck="false"
         t-on-keydown="onKeydown"
-        t-on-mousedown="onMousedown"
+        t-on-mousedown.stop="onMousedown"
+        t-on-mouseup.stop=""
+        t-on-mousewheel.stop=""
```

Comment:
Maybe we can stop the wheel event only if the composer have a scrollbar. probably not worth it tho.